common/ripple/user_utils.py

```diff
@@ -700,37 +686,51 @@ async def grant_user_default_privileges(user_id: int) -> None:
     )
 
 
-async def authorize_login_and_activate_new_account(
+WINE_STATIC_MAC_ADDRESS = "b4ec3c4334a0249dae95c284ec5983df"
+WINE_STATIC_DISK_ID = "ffae06fb022871fe9beb58b005c5e21d"
+
+
+def _hwid_set_running_under_wine(hwid_set: list[str]) -> bool:
+    return hwid_set[2] == WINE_STATIC_MAC_ADDRESS or hwid_set[4] == WINE_STATIC_DISK_ID
+
+
+async def find_user_ids_with_hwid_matches(
     user_id: int,
-    # TODO: refactor hwid sets into an object across the codebase
     hwid_set: list[str],

```

Comment:
TODO: big goal of this PR will be to encapsulate device information in a more human-friendly format